packages/backend/src/controllers/rateLimiter.ts

```diff
@@ -0,0 +1,12 @@
+import { RequestHandler } from 'express';
+import { rateLimit } from 'express-rate-limit';
+
+export const limiter = rateLimit({
+    windowMs: 60 * 1000, // 1 minute
+    limit: 100, // Limit each IP to 100 requests per `window` (per minute).
```

Comment:
is this a good limit ? it seems too much for this single endpoint, but at the same time, we should be able to reuse this limit for other endpoints